src/views/List.jsx

```diff
@@ -49,5 +55,10 @@ export function List({ data, listToken }) {
 				))}
 			</ul>
 		</>
+	) : (
+		<>
+			<p>The shopping list is currently empty</p>
+			<button onClick={handleAddItem}>Add Item</button>
```

Comment:
Was just curious on wording here, its good but was wondering if it could be more welcoming? For example maybe "Would you like to add an item?" above the button? Not necessary but just something I thought about. 

Also I know we didn't do this with all our buttons bout would labels be good for buttons as well for accessibility? Like aria labels? Maybe not or also something we could always add later. Overall looks great!